packages/hoppscotch-common/src/components/teams/Edit.vue

```diff
@@ -35,12 +35,12 @@
           v-if="
             !teamDetails.loading &&
             E.isRight(teamDetails.data) &&
-            teamDetails.data.right.team.teamMembers
+            teamDetails.data.right.team?.teamMembers
```

Comment:
We can get rid of this condition from the parent. Safeguards can be added for the child elements. It already exists for the list render block.

https://github.com/hoppscotch/hoppscotch/blob/73b3ff8e411705dd842abb3088b1a20bc9aa7df5/packages/hoppscotch-common/src/components/teams/Edit.vue#L344